Wangscape/noise/module/ModulePtr.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <memory>
+#include "WrapperBase.h"
+
+namespace noise
+{
+namespace module
+{
+
+typedef std::shared_ptr<WrapperBase> ModulePtr;
```

Comment:
Once the `Reseedable*` classes are removed, all modules used in Wangscape will be contained in a `Wrapper`. The exceptions will be:
*`RasterBase::build` takes a `Module&` parameter (this can be changed).
* `Module::SetSourceModule`, etc. take a `Module&` parameter.
* `Module::m_pSourceModule` internally uses a raw `Module**`.
I think that's enough to justify using the name `ModulePtr`, but it can be changed to `WrapperPtr` if necessary.